packages/otelbin/src/lib/metadataUtils.ts

```diff
@@ -0,0 +1,67 @@
+// SPDX-FileCopyrightText: 2023 Dash0 Inc.
```

Comment:
**Suggestion:** Put this file next to `social-preview/[id]` to put related code closer together.